homeassistant/components/playstation_network/binary_sensor.py

```diff
@@ -0,0 +1,108 @@
+"""Binary Sensor platform for PlayStation Network integration."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from enum import StrEnum
+from typing import TYPE_CHECKING
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import (
+    PlaystationNetworkConfigEntry,
+    PlaystationNetworkCoordinator,
+    PlaystationNetworkData,
+)
+
+PARALLEL_UPDATES = 0
+
+
+@dataclass(kw_only=True, frozen=True)
+class PlaystationNetworkBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """PlayStation Network binary sensor description."""
+
+    value_fn: Callable[[PlaystationNetworkData], StateType]
+
+
+class PlaystationNetworkBinarySensor(StrEnum):
+    """PlayStation Network binary sensors."""
+
+    ONLINE_STATUS = "online_status"
+    PS_PLUS_STATUS = "ps_plus_status"
+
+
+BINARY_SENSOR_DESCRIPTIONS: tuple[
+    PlaystationNetworkBinarySensorEntityDescription, ...
+] = (
+    PlaystationNetworkBinarySensorEntityDescription(
+        key=PlaystationNetworkBinarySensor.ONLINE_STATUS,
+        translation_key=PlaystationNetworkBinarySensor.ONLINE_STATUS,
+        name="Online",
+        value_fn=(lambda psn: psn.available),
```

Comment:
```suggestion
        value_fn=lambda psn: psn.available,
```
Only wrap if they span more than one line